### ISSUE TITLE
work_mem when max_worker_processes used

### ISSUE THREAD

**Issue by wardal** [le0pard/pgtune#26]
Hey Leopard,

Not sure you correctly count work_mem when max_worker_processes used.

From your formula, as I understood, you set:
`max_parallel_workers_per_gather = max_worker_processes / 2`, so for example,
if I set 16 workers, it will count 8 parallel workers.

So, for example, when I don't set numbers of CPUs, in results you propose to set work_mem to 100MB (for example). But if I set 4 CPU what mean in your script:
```
max_worker_processes = 4
max_parallel_workers_per_gather = 2
max_parallel_workers = 4
```
you will show  work_mem = 50MB, because you count it as (`work_mem / max_parallel_workers_per_gather`)

In other words with `max_parallel_workers_per_gather = 2` in your formula you decrease work_mem for 50%, but in Postgresql `max_parallel_workers_per_gather = 2` is default value, so with or without this value in your script it must show the same work_mem.

**Comment by le0pard** [comment]
Hello @wardal Just to be clear: do you mean it is incorrect, because of default value, which pgtune don't know? Thanks.

**Comment by le0pard** [comment]
@wardal fixed by https://github.com/le0pard/pgtune/commit/8fc6acdc37c2ce7c215fa8b3e407eab79ceefb97
